librz/core/cmd_debug.c

```diff
@@ -1443,370 +1443,618 @@ static bool get_bin_info(RzCore *core, const char *file, ut64 baseaddr, PJ *pj,
 	return true;
 }
 
-static int cmd_debug_map(RzCore *core, const char *input) {
+// dm
+RZ_IPI RzCmdStatus rz_cmd_debug_list_maps_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	if (argc == 1) {
+		char str[5] = "";
+		if (argc == 1) {
+			strcpy(str, argv[0] + 2);
```

Comment:
Are you sure `strcpy()` is necessary here? Maybe it will work without?

Reply:
yeah it is not needed ill remove it